s3prl/downstream/a2o-vc-vcc2020/batch_vc_decode.sh

```diff
@@ -30,11 +31,17 @@ fi
 
 for trgspk in "${trgspks[@]}"; do
     for ep in $(seq ${start_ep} ${interval} ${end_ep}); do
-        echo "Objective evaluation: Ep ${ep}; trgspk ${trgspk}"
+        echo "Objective evaluation: Upstream ${upstream}, Ep ${ep}; trgspk ${trgspk}"
         expname=a2o_vc_vcc2020_${tag}_${trgspk}_${upstream}
-        expdir=../../result/downstream/${expname}
-        ./decode.sh pwg_${task}/ ${expdir}/${ep} ${trgspk}
+        expdir=result/downstream/${expname}
```

Comment:
Is this `expdir` a typo of `../result/downstream/${expname}` ?
I thought there is no expdir under the task directory.